special-pages/pages/new-tab/app/next-steps/nextsteps.translations.json

```diff
@@ -0,0 +1,1082 @@
+{
```

Comment:
This won't be used right away, and is unfortunately not in the correct format (for us) - we'll do 1 big submission soon, but you can remove this file.